brainkeeper/src/app/components/session_screen/grid/grid.component.ts

```diff
@@ -1,15 +1,42 @@
 import { Component, OnInit } from '@angular/core';
 
+
+import { SessionService } from 'src/app/services/session-service';
+
 @Component({
   selector: 'app-grid',
   templateUrl: './grid.component.html',
   styleUrls: ['./grid.component.scss']
 })
 export class GridComponent implements OnInit {
 
-  constructor() { }
+  public personNames: string[];
```

Comment:
I think further processing would benefit from a single person array here and not two string arrays.

Reply:
Changed with commit 46c0046